Cargo.toml

```diff
@@ -29,7 +31,8 @@ serde_json = "1"
 tokio = { version = "1.18", features = ["full"] }
 toml = "0.5"
 uint = { version = "0.9.3", default-features = false }
-workspaces = { git = "https://github.com/near/workspaces-rs.git", rev = "8b3356a29f71756d86840679d3ff162b885123c9" }
+# Feature `unstable` is used to enable `compile_project()`.
+workspaces = { git = "https://github.com/near/workspaces-rs.git", rev = "8b3356a29f71756d86840679d3ff162b885123c9", features = ["unstable"] }
```

Comment:
Can we use the newer version of workspaces already?

Reply:
For the acl test: yes! I think overall using `0.6` is fine since CI in #21 passed and `0.6` is ahead only two commits of the fix we previously targeted.